instrumentation/opentelemetry-instrumentation-boto3sqs/src/opentelemetry/instrumentation/boto3sqs/__init__.py

```diff
@@ -0,0 +1,428 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Instrument `boto3sqs`_ to trace SQS applications.
+
+.. _boto3sqs: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html
+
+
+Usage
+-----
+
+.. code:: python
+
+    import boto3
+    from opentelemetry.instrumentation.boto3sqs import Boto3SQSInstrumentor
+
+
+    Boto3SQSInstrumentor().instrument()
+"""
+import logging
+from typing import Any, Collection, Dict, Generator, List, Optional
+
+import boto3
+import botocore.client
+import wrapt
```

Comment:
`boto3` and `wrapt` should be dependencies of this package.

Reply:
For consistency with the rest of imports, please use `from ... import` whenever possible.

Reply:
I have added wrapt, but isn't boto3 added as part of `_instruments` from `package.py`?